certs/ca.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpQIBAAKCAQEAvvEsdX6LuXQVEPoXCVmXwQurHGAa1twpwUeTVjqHT3ZFjj+R
+1+LBLS2ZRg7sN8ss8nNNWFpvcVCpQn35wEHejK1OIuMdqFCpNjGH4xq1XfMASugg
+5W5Jdy0HGovv0qBmtltVUGnR8DncWg0Ll83oecrC7KlyuLrMU8phBE3cuJ3H2RKY
+Fe0NVln68k61svNHRRn/Q8Z1N9yXN2MmXt0Qh1CT+87SvDXNV0zzYKcLvgNyp3TU
+Z0cwdfM5IwrhRXWlPx5N6jStedPCB8+NhLurMXvDymP91fLjZZWCHa3+YWloR4Ej
+VpEoQ1UciP7FiZMthVVptHdfJtNoJZIu8vR51QIDAQABAoIBAQCGcYgNXsIBHbQr
+lPwdcEEDAzVDMIQVX0XjOoOKhoF5uNw20ZneOWItblrq0l0O1/dY60i9ekfXsKTN
+FaiB759EzE4lEGd0eP78EpoMlWS4tv4wwAtRoHsH9+N1lKyTUK+hnOOQUEIq9IKr
+wV21UJ1tDBB91L3bRmPSZ8Ax6KUOM4pmSayo6zI1t8xkbppEMsJv0y5oXmnjVF+y
+wvO+HjbR6vkttPF9atc0TQgL0T9aSccpQLmbUAqCah3hAxISXXR9KW9ALtzIzjxI
+iuQK0Dk04bM+8v1wf4eCGzpmea17XdsvEv3D/+zu51Mn0rLoH945SStqFM1Had/H
+ZYhIvLONAoGBAOuffLm9oIfoDW/5M6UUabiYx/AKtZCxRNYtNf09hPZsKSTj0J2C
```

Comment:
Since, we needed the cert directory for the gen_cert.sh script, so I decided to keep this ca.key instead .gitkeep

Reply:
This key is generated from openssl and is used in validation webhook configuration [here](https://github.com/casbin/k8s-authz/blob/b5e67d7e91d46bbc105dc81a2441b2734a8126fc/manifests/webhook.yaml#L12). Sorry for commiting it, It's not safe to put on Github, but since it was generated only for sample usage, it won't bother anyone. I would simply replace it with a .gitgnore file in my next commit.